plugins/dune-rules/test/test.t

```diff
@@ -0,0 +1,134 @@
+This file test the behaviour of the `qcheck-stm` sub-command of the `dune` plugin.
+
+  $ export ORTAC_PLUGINS=qcheck-stm,dune-rules
```

Comment:
This should be `ORTAC_ONLY_PLUGIN=dune-rules`, I suppose.

Reply:
It is in fact useless as it is clear from the rule that is generated which plugin should be loaded.
Thanks for spotting this.